src/Traits/Form/FormControl.php

```diff
@@ -94,11 +94,14 @@ public function getValue()
      */
     public function setValue($value = null)
     {
-        if (is_callable($value)) {
+        if (is_callable($value) && !is_string($value)) {
```

Comment:
@HAYASHI-Masayuki don't you think it's better to ensure the $value is an instance of Closure instead?

Reply:
Thank you for advice.

Is callable array allowed here?
If it is not allowed, it will be more approciate.

Sorry, I don't have enough knowledge of this codebase.

Reply:
actually, it was..., but I've never seen the real reason to use it here... I think, accepting the value as a closure is predictable and more than enough to simply set a value.

Reply:
Thanks. I modified code.